gradio/__init__.py

```diff
@@ -104,7 +106,4 @@
 )
 from gradio.themes import Base as Theme
 
-current_pkg_version = (
-    (pkgutil.get_data(__name__, "version.txt") or b"").decode("ascii").strip()
-)
-__version__ = current_pkg_version
```

Comment:
We should keep `__version__` as doing `gradio.__version__` is the canonical way to get the version of a package